schemas/logs/git_lab/api.yml

```diff
@@ -21,34 +18,13 @@ fields:
   - name: duration_s
     required: true
     description: The time spent serving the request (in seconds)
-    type: object
```

Comment:
This is problematic - it will break tables. What is the migration plan here?

Reply:
See https://github.com/panther-labs/panther-analysis/pull/229#discussion_r604046134